core/src/main/java/com/cyberbot/bomberman/core/models/net/packets/control.kt

```diff
@@ -0,0 +1,50 @@
+package com.cyberbot.bomberman.core.models.net.packets
+
+import com.cyberbot.bomberman.core.models.net.data.PlayerData
+import com.google.gson.GsonBuilder
+import com.google.gson.typeadapters.RuntimeTypeAdapterFactory
+
+private const val TYPE = "type"
+
+open class ControlPacket
+
+class Client(val id: Long? = null, val nick: String? = null)
+class Lobby(val id: String? = null, val owner: Client? = null, val clients: MutableList<Client> = ArrayList())
```

Comment:
Keeping owner client object in ArrayList with other clients and storing only owners id in owner field would allow for easier lobby ownership transfer,  if such feature were to be implemented

Reply:
Good point, it also allows for less data transfer